sdk/python/tests/run_tests.sh

```diff
@@ -13,13 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
 cd $(dirname $0)/..
-rm -rf .testing-env
-python3 -m venv .testing-env && \
-source .testing-env/bin/activate && \
 python3 -m pip install -U pip
 python3 -m pip install pytest
 # install local kubeflow.metadata package
 python3 -m pip install -e .
-python3 -m pytest tests/test.py
-rm -rf .testing-env
+python3 -m pytest ./tests
+# Run integration tests multiple times should get the same result. This checks
+# the idenpotence of the tests.
+echo "Run tests the second time:"
+python3 -m pytest ./tests
```

Comment:
sorry, confused me, how to check the idenpotence for the twice running here? or you want to test reuse case if existing, no need check, just ensure case pass?

Reply:
Context: one test here is almost the same as the sample notebook. It is ideal that multiple notebook runs continue to work.

Sorry, idempotency is not the right term here, but these tests now update and depend on DB states. Just want to make sure each individual test continue working even if the DB has previous run state.

Reply:
Oh.. Make sense then, thanks @zhenghuiwang